ddtrace/llmobs/_integrations/bedrock.py

```diff
@@ -55,3 +55,39 @@ def _llmobs_metrics(span: Span, formatted_response: Optional[Dict[str, Any]]) ->
             metrics["completion_tokens"] = completion_tokens
             metrics["total_tokens"] = prompt_tokens + completion_tokens
         return metrics
+
+    @staticmethod
+    def _extract_input_message(prompt):
+        """Extract input messages from the stored prompt.
+        Anthropic allows for messages and multiple texts in a message, which requires some special casing.
+        """
+        if isinstance(prompt, str):
+            return [{"content": prompt}]
+        if not isinstance(prompt, list):
```

Comment:
Is it conventional for the python tracer to add a warning log in cases like this?

Might be helpful for users if the spec for prompt changes again and their data suddenty doesn't show up

Reply:
Good point, will add a warning